chimg/chroot.py

```diff
@@ -112,7 +112,8 @@ def _snaps_base_install(self, snap_infos: Dict[str, SnapInfo]) -> None:
             required_cores.add(core_name)
 
         for core in required_cores:
-            self._snap_install(name=core, channel="stable")
+            snap_infos[core ] = self._snap_install(name=core, channel="stable")
```

Comment:
extra space

Reply:
I see you fix it in a latter commit, you should fix it in this commit.

Reply:
fixed